test/specs/UnorganizedTest.js

```diff
@@ -1,82 +1,66 @@
-describe("Search for the first discount skincare product and check the price", () => {
-  before(() => {
-    browser.url("");
-    browser.pause(1000);
-    browser.maximizeWindow();
-  });
+const { HomePage, DiscountPage, ItemPage, CheckoutPage } = require('../pageObjects');
 
-  it("should navigate to discount page", () => {
-    const discounts = $(".specials");
-    discounts.click();
-    expect(browser.getUrl()).toContain("aktsiyi");
+describe('Search for the first discount skincare product and check the price', () => {
+  it('should setup before test scenario', () => {
+    HomePage.navigate();
+    HomePage.waitLoaded();
+    HomePage.agreeToComplianceBtn.click();
+    HomePage.closeBanner();
   });
 
-  it("should navigate to specific category", () => {
-    const closeCookies = $(".close");
-    closeCookies.click();
-    const category = $('//*[@id="param-filter"]/div[1]/div[2]/div[6]');
-    category.scrollIntoView();
-    category.click();
-    expect($(".category-tree__item--active").getText()).toEqual(
-      "Косметика для обличчя"
-    );
+  it('should navigate to discount page', () => {
+    HomePage.discountSection.click();
+    DiscountPage.waitLoaded();
+    expect(DiscountPage.url).toContain('aktsiyi');
   });
 
-  it("should limit the Max Price", () => {
-    const maxPrice = $(".slider-input-max");
-    maxPrice.scrollIntoView();
-    maxPrice.click();
-    maxPrice.setValue("500");
-    expect(maxPrice.getValue()).toEqual("500");
+  it('should limit the Max Price', () => {
+    DiscountPage.maxPrice.scrollIntoView();
+    DiscountPage.maxPrice.click();
+    DiscountPage.setMaxPrice('500');
+    expect(DiscountPage.maxPrice.getValue()).toEqual('500');
   });
 
-  it("should limit the Min Price", () => {
-    const maxPrice = $(".slider-input-min");
-    maxPrice.scrollIntoView();
-    maxPrice.click();
-    maxPrice.setValue("0");
-    expect(maxPrice.getValue()).toEqual("0");
+  it('should limit the Min Price', () => {
+    DiscountPage.minPrice.scrollIntoView();
+    DiscountPage.minPrice.click();
+    DiscountPage.setMinPrice('0');
+    expect(DiscountPage.minPrice.getValue()).toEqual('0');
   });
 
-  xit("should select only high-rated items", async () => {
-    await browser.pause(1000);
-    const highestRateCheckbox = await $('[title="Оцінка 5*"]');
-    highestRateCheckbox.scrollIntoView();
-    await highestRateCheckbox.click();
-    await highestRateCheckbox.scrollIntoView();
-    if (await (await $(".exponea-close-cross")).isDisplayed()) {
-      await (await $(".exponea-close-cross")).click();
+  it('should select only high-rated items', () => {
+    browser.pause(1000);
```

Comment:
Try to avoid using pause. Instead use more fluent waits. If operator can be moved inside page objects.